### PR TITLE
Updated chains list

### PR DESCRIPTION
**Pull request by stackman27**

### CODE REVIEW

File: selectors.yml [hunk 1]
```diff
@@ -107,18 +146,30 @@ selectors:
   421614:
     selector: 3478487238524512106
     name: "ethereum-testnet-sepolia-arbitrum-1"
+  432201:
+    selector: 1458281248224512906 
+    name: "dexalot-subnet-testnet"
```
Comment (andrevmatos) [inline selectors.yml:151]: subnet-testnet? :thinking:

File: selectors.yml [hunk 2]
```diff
@@ -3,18 +3,36 @@ selectors:
...
-    name: "ethereum-mainnet-xlayer-1"
+  250: 
+    selector: 3768048213127883732
+    name: "fantom-testnet-opera" 
   280:
     selector: 6802309497652714138
     name: "ethereum-testnet-goerli-zksync-1"
+  282: 
+    selector: 3842103497652714138 
+    name: "cronos-testnet-zkevm"
```
Comment (andrevmatos) [inline selectors.yml:35]: zkevm is the name of their testnet?
looks like a zk-rollup on top of their (unnamed) testnet; if so, it should be named
```suggestion
    name: "cronos-testnet-zkevm-1"
```

File: selectors.yml [hunk 3]
```diff
@@ -24,9 +42,15 @@ selectors:
...
     name: "cronos-testnet"
+  398:
+    selector: 5061593697262339000
+    name: near-testnet
   420:
     selector: 2664363617261496610
     name: "ethereum-testnet-goerli-optimism-1"
+  462:
+    selector: 7317911323415911000
+    name: areon-network-testnet
```
Comment (andrevmatos) [inline selectors.yml:53]: Maybe for consistency with `gnosis_chain` and friends?
Or even drop the `_network` entirely, as most projects here do have network in their names (unless there's precedent deployments using that name)
```suggestion
    name: areon_network-testnet
```

File: selectors.yml [hunk 4]
```diff
@@ -3,18 +3,36 @@ selectors:
...
     name: "binance_smart_chain-testnet"
+  111: 
+    selector: 572210378683744374 
+    name: "velas-testnet"
   195:
     selector: 2066098519157881736
     name: "ethereum-testnet-sepolia-xlayer-1"
-  196:
-    selector: 3016212468291539606
-    name: "ethereum-mainnet-xlayer-1"
```
Comment (andrevmatos) [inline selectors.yml:14]: Maybe better not remove names which were already committed?

File: selectors.yml [hunk 5]
```diff
@@ -107,18 +146,30 @@ selectors:
...
     name: "ethereum-testnet-sepolia-arbitrum-1"
+  432201:
+    selector: 1458281248224512906 
+    name: "dexalot-subnet-testnet"
   534351:
     selector: 2279865765895943307
     name: "ethereum-testnet-sepolia-scroll-1"
+  686868:
+    selector: 5269261765892944301 
+    name: "bitcoin-merlin-testnet"
```
Comment (andrevmatos) [inline selectors.yml:157]: the project/network is called `bitcoin-merlin`? Or is `merlin` the name of the bitcoin testnet, in which case it should be named ?
```suggestion
    name: "bitcoin-testnet-merlin"
```

File: selectors.yml [hunk 6]
```diff
@@ -184,3 +295,6 @@ selectors:
   534352:
     selector: 13204309965629103672
     name: "ethereum-mainnet-scroll-1"
+  810180: 
+    selector: 4350319965322101699
+    name: "zklink-nova-mainnet"
```
Comment (andrevmatos) [inline selectors.yml:300]: maybe?
```suggestion
    name: "zklink_nova-mainnet" 
```

File: selectors.yml (hunk 1 above)
Comment (stackman27) [reply selectors.yml:151]: yea the subnet has a testnet. https://subnets-test.avax.network/dexalot

File: selectors.yml (hunk 4 above)
Comment (stackman27) [reply selectors.yml:14]: i placed it in the mainnet section. It was placed in the testnet section 
https://github.com/smartcontractkit/chain-selectors/pull/40/files#diff-6724bdfc10769b3c4b08cdf2c057660d7ac98cedf4c3900fb5be959e034f84fbR209-R210

File: selectors.yml (hunk 1 above)
Comment (andrevmatos) [reply selectors.yml:157]: Interesting. Is this an avalanche subnet? I don't think we have prior art for this kind of stuff on naming proposal; should we come up with something?
How about `avalanche-subnet-dexalot-testnet`? i.e. the testnet of the dexalot subnet attached to avalanche (like testnets and rollups)

Comment (makramkd) [reply selectors.yml:157]: Why not follow RDD naming convention instead of creating a new one?

File: selectors.yml [hunk 7]
```diff
@@ -3,18 +3,36 @@ selectors:
...
     name: "bitcoin-testnet-rootstock"
+  41:
+    selector: 729797994450396300
+    name: telos-evm-testnet
+  45:
+    selector: 4340886533089894000
+    name: darwinia-testnet-pangoro
+  53:
+    selector: 8955032871639343000
+    name: coinex_smart_chain-testnet
```
Comment (makramkd) [inline selectors.yml:14]: These `name` fields are not surrounded by double-quotes

File: selectors.yml [hunk 8]
```diff
@@ -24,9 +42,15 @@ selectors:
...
     name: "cronos-testnet"
+  398:
+    selector: 5061593697262339000
+    name: near-testnet
   420:
     selector: 2664363617261496610
     name: "ethereum-testnet-goerli-optimism-1"
+  462:
+    selector: 7317911323415911000
+    name: areon-testnet
```
Comment (makramkd) [inline selectors.yml:53]: Quote the names

File: selectors.yml [hunk 9]
```diff
@@ -35,13 +59,19 @@ selectors:
...
+    name: "moonbeam-testnet-moonbase-alpha"
   1337:
     selector: 3379446385462418246
     name: "geth-testnet"
   1442:
     selector: 11059667695644972511
     name: "ethereum-testnet-goerli-polygon-zkevm-1"
+  1908:
+    selector: 4888058894222120000
+    name: bitcichain-testnet
```
Comment (makramkd) [inline selectors.yml:74]: Quote

File: selectors.yml [hunk 10]
```diff
@@ -66,12 +102,21 @@ selectors:
   5003:
     selector: 8236463271206331221
     name: "ethereum-testnet-sepolia-mantle-1"
+  9559:
+    selector: 1113014352258747600
+    name: neonlink-testnet
```
Comment (makramkd) [inline selectors.yml:107]: Quote

File: selectors.yml [hunk 11]
```diff
@@ -130,21 +187,57 @@ selectors:
...
+    name: "cronos-mainnet"
+  40:
+    selector: 1477345371608778000
+    name: telos-evm-mainnet
+  46:
+    selector: 8866418665544333000
+    name: darwinia-mainnet
+  52:
+    selector: 1761333065194157300
+    name: coinex_smart_chain-mainnet
```
Comment (makramkd) [inline selectors.yml:201]: Quotes

File: selectors.yml [hunk 12]
```diff
@@ -130,21 +187,57 @@ selectors:
...
+    name: "ethereum-mainnet-zksync-1" 
+  397:
+    selector: 2039744413822257700
+    name: near-mainnet
+  463:
+    selector: 1939936305787790600
+    name: areon-mainnet
+  592: 
+    selector: 6422105447186081193 
+    name: "astar-mainnet"
```
Comment (makramkd) [inline selectors.yml:240]: Quotes

**Review by makramkd — CHANGES_REQUESTED** [review]
`name` fields are inconsistently double-quoted, lets double-quote all of them since that seems to be the convention.

File: selectors.yml (hunk 1 above)
Comment (andrevmatos) [reply selectors.yml:157]: @makramkd that's the [whole idea of this repo](https://smartcontract-it.atlassian.net/wiki/spaces/DEPLOY/pages/599491447/Network+naming+proposal). But there's some room for decision still on that document (e.g. it doesn't anticipate these subnets), and as such we need to find the schema which best matches the "spirit" of the whole proposal.

Comment (makramkd) [reply selectors.yml:157]: FWIW I think avalanche-subnet-dexalot-testnet matches RDD naming convention, but would be good to get engop's review on this.

File: selectors.yml [hunk 13]
```diff
@@ -107,18 +152,30 @@ selectors:
...
     name: "ethereum-testnet-sepolia-scroll-1"
+  686868:
+    selector: 5269261765892944301 
+    name: "bitcoin-testnet-merlin"
   595581:
     selector: 7837562506228496256
     name: "avalanche-testnet-nexon"
+  810181: 
+    selector: 5837261596322416298
+    name: "zklink-nova-testnet"
```
Comment (stackman27) [inline selectors.yml:169]: Note: zkLink-nova is an L3 "zkLink Nova is the industry’s first aggregated Layer 3 zkEVM Rollup network built on top of Ethereum and Ethereum Layer 2 rollups (L2s)." 

not sure how to go about it. Should it be "ethereum-testnet-"layer2Chain"-testnet-zklink-nova"?

File: selectors.yml (hunk 1 above)
Comment (stackman27) [reply selectors.yml]: added!

File: selectors.yml [hunk 14]
```diff
@@ -184,3 +301,6 @@ selectors:
   534352:
     selector: 13204309965629103672
     name: "ethereum-mainnet-scroll-1"
+  810180: 
+    selector: 4350319965322101699
+    name: "zklink_nova-mainnet"
```
Comment (stackman27) [inline selectors.yml:306]: same question as above: https://github.com/smartcontractkit/chain-selectors/pull/40/files#r1626472810

File: selectors.yml (hunk 13 above)
Comment (makramkd) [reply selectors.yml:169]: Please reach out to engops directly, don't think they're getting notified on this PR

**Review by polds — DISMISSED** [review]

File: selectors.yml (hunk 13 above)
Comment (andrevmatos) [inline selectors.yml:169]: For consistency with zklink_nova-mainnet:
```suggestion
    name: "zklink_nova-testnet" 
```

**Review by andrevmatos — COMMENTED** [review]
I've checked the project, and they don't have a specific chain where they are anchored, and instead seems to settle on multiple L2 and Ethereum L1 jointly. Therefore, I think it deserves to be a top-level network, even though they call themselves a L3.
Please, just fixup testnet name, and we can get this merged as is. Thank you for your patience, naming is hard.

**Review by andrevmatos — DISMISSED** [review]

File: selectors.yml [hunk 15]
```diff
@@ -107,18 +152,30 @@ selectors:
...
+    name: "zklink_nova-testnet" 
   11155111:
     selector: 16015286601757825753
     name: "ethereum-testnet-sepolia"
   11155420:
     selector: 5224473277236331295
     name: "ethereum-testnet-sepolia-optimism-1"
+  21000000:
+    selector: 1467427327723633929
+    name: "ethereum-testnet-corn-1"
```
Comment (andrevmatos) [inline selectors.yml:178]: Is this a L1 ethereum-testnet?

File: selectors.yml [hunk 16]
```diff
@@ -130,21 +187,57 @@ selectors:
...
     name: "polygon-mainnet"
+  196:
+    selector: 3016212468291539606
+    name: "ethereum-mainnet-xlayer-1"
   199:
     selector: 3776006016387883143
     name: "bittorrent_chain-mainnet"
+  252:
+    selector: 1462016016387883143 
+    name: "fraxtal-mainnet"
```
Comment (andrevmatos) [inline selectors.yml:222]: Is this a true L1? Saw they're a testnet rollup on sepolia

File: selectors.yml [hunk 17]
```diff
@@ -130,21 +187,57 @@ selectors:
...
+    name: "ethereum-mainnet-zksync-1" 
+  397:
+    selector: 2039744413822257700
+    name: "near-mainnet"
+  463:
+    selector: 1939936305787790600
+    name: "areon-mainnet"
+  592: 
+    selector: 6422105447186081193 
+    name: "polkadot-mainnet-astar"
```
Comment (andrevmatos) [inline selectors.yml:240]: What? They have multiple mainnets?

**Review by andrevmatos — APPROVED** [review]

File: selectors.yml (hunk 17 above)
Comment (stackman27) [reply selectors.yml:240]: Those are 2 different chains. One is zkEVM, the other isn't

**Pull request merged**